.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,15 +1,20 @@
+#### 📁 Related issues
+
 <!-- 
-Things to check before submitting a pull request:
-  * Label this PR with the correct semver label (if you have permission to do so).
-    - semver.patch: Backwards compatible bug fixes.
-    - semver.minor: Backwards compatible feature additions.
-    - semver.major: Breaking changes. This includes changing interfaces or configuration behaviour.
-  * Target the correct branch. Patch updates can target main, other changes should target the latest versions/* branch.
-  * Update the RELEASE_NOTES.md document in case of relevant feature or config changes.
+Reference any relevant issues here. Closing keywords only have an effect when targeting the main branch. If there are no related issues, you must first create an issue through https://github.com/solid/community-server/issues/new/choose
 -->
 
-#### Related issues
-<!-- Reference any relevant issues here. Closing keywords only have an effect when targeting the main branch. -->
+#### ✍️ Description
 
-#### Description
 <!-- Describe the relevant changes in this PR. Also add notes that might be relevant for code reviewers. -->
+
+
+### ✅ PR check list
+
+Before this pull request can be merged, a core maintainer will check whether
```

Comment:
I would be more explicit that we do expect this to already be ok when the PR is done (except the label)

Reply:
Well, a core maintainer will check whether it’s done, but if you don’t know how to do this, I don’t think it should block you from opening the PR. What do you think?